test/client/nodes_test.rb

```diff
@@ -41,7 +41,7 @@
   it "gets the hot threads for the node(s)" do
     str = $client.nodes.hot_threads :read_timeout => 2
     assert_instance_of String, str
-    assert_match %r/cpu usage by thread/, str
+    assert_match %r/:::/, str
```

Comment:
:+1: to HTTP OK **and** checking for a non-empty response body. We really don't care what is returned just so long as *something* is returned.